package/system/uci/Makefile

```diff
@@ -67,6 +84,11 @@ define Package/libuci-lua/install
 	$(CP) $(PKG_BUILD_DIR)/lua/uci.so $(1)/usr/lib/lua/
 endef
 
+define Package/libuci-lua53/install
+	$(INSTALL_DIR) $(1)/usr/local/lib/lua/5.3
+	$(CP) $(PKG_BUILD_DIR)/lua/uci.so $(1)/usr/local/lib/lua/5.3
```

Comment:
Please do not install this to `/local`.
Is it working when you install it into `/usr/lib/lua/5.3/` instead?

Reply:
The default prefix is `/usr/local` in Lua5.3.
There are some packages already use it such as luasocket-lua-53.

Reply:
ok then this is fine.